css/style.css

```diff
@@ -0,0 +1,17 @@
+/** @format */
```

Comment:
If you add a comment don't add extra * to the comment

Reply:
- [ ] Kindly phrase your comments politely, as suggestions instead of orders.
`HINT: * This is a polite suggestion: *"I think it would be better to don't add the comment block"*`
`* This reads as an order: *"Don't add comment"*`
- [ ] Also make sure you always include an explanation, a "why", for each requested change.

Reply:
I think it would be better if you use a single * after the / when you're commenting your css code.
`/* @format */`
I would like to recommend you keep it that way on your further comments.

Reply:
I think it would be better if you use a single * after the / when you're commenting your css code.
`/* @format */`
I would like to recommend you keep it that way on your further comments, so it can make your code look stylish.